libs/@guardian/source-development-kitchen/src/react-components/ticker/Ticker.tsx

```diff
@@ -18,6 +18,7 @@ interface TickerCopy {
 export interface TickerData {
 	total: number;
 	goal: number;
+	goalCopy?: string;
```

Comment:
I think this belongs in the `TickerCopy` object

Reply:
ok, I will change it to tickerCopy.